### PR TITLE
[INLONG-8169][Manager] Add default tenant "public" if the request does not specify

### LINKED ISSUE #8169: [Feature][Manager] Add default tenant "public" if the request does not specify one.
### Description

To support multi-tenancy #7914, all requests need to carry tenant fields in header.
To be compitable with old version, if the requst does not, use "public" tenant.

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by vernedeng**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #8169 
- Parent #7914 

### Motivation

To support multi-tenancy https://github.com/apache/inlong/issues/7914, all requests need to carry tenant fields in header.
To be compitable with old version, if the requst does not, use "public" tenant.

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [x] This change added tests and can be verified as follows: EmptyTenantFilterTest

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

File: inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/filter/EmptyTenantFilter.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+
+import java.io.IOException;
+
+import static org.apache.inlong.common.util.BasicAuth.BASIC_AUTH_TENANT_HEADER;
+import static org.apache.inlong.common.util.BasicAuth.DEFAULT_TENANT;
+
+public class EmptyTenantFilter implements Filter {
```
Comment (healchow) [inline inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/filter/EmptyTenantFilter.java:36]: Please add javadoc for the class, thanks.

File: inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/filter/HttpServletRequestFilter.java [hunk 2]
```diff
@@ -38,9 +35,6 @@
  * Make All
  */
 @Slf4j
-@Component
-@WebFilter
-@Order(0)
```
Comment (healchow) [inline inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/filter/HttpServletRequestFilter.java:43]: Why remove those annotations?

Comment (vernedeng) [reply inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/filter/HttpServletRequestFilter.java:43]: All web filter will be managed in WebFilterConfig

File: inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/filter/EmptyTenantFilter.java (hunk 1 above)
Comment (vernedeng) [reply inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/filter/EmptyTenantFilter.java:40]: fixed, thx

**Review by healchow — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**